PowerShell/Deploy/Build-Module.ps1

```diff
@@ -48,4 +67,13 @@ $NewModuleChangelogRecord = New-ModuleChangelog -LatestVersion:($ModuleVersion)
 If (!(($ModuleChangelog | Select-Object -First 1) -match $ModuleVersion)) {
     ($NewModuleChangelogRecord + ($ModuleChangelog | Out-String)).Trim() | Set-Content -Path:($FilePath_ModuleChangelog) -Force
 }
-# EndRegion Updating module change log
\ No newline at end of file
+# EndRegion Updating module change log
+
+Write-Host "Building help files" -ForegroundColor Green
+."$PSScriptRoot/Build-HelpFiles.ps1" -ModuleName "JumpCloud" -ModulePath "./PowerShell/JumpCloud Module"
```

Comment:
```suggestion
."$PSScriptRoot/Build-HelpFiles.ps1" -ModuleName "JumpCloud" -ModulePath (Get-Item -Path:($FilePath_psd1)).directory
```

Reply:
Depending on where you call this function, It'll throw errors, I want this to be as simple as possible, if we sub a variable for a hard-coded path, it'll be more likely to work when called from different working dirs.